internal/utils/cache-dir.go

```diff
@@ -0,0 +1,25 @@
+package utils
+
+import (
+	"os"
+	"path"
+)
+
+func SnykCacheDir() (string, error) {
+	baseDirectory, err := os.UserCacheDir()
+	if err != nil {
+		return "", err
+	}
+
+	snykCacheDir := path.Join(baseDirectory, "snyk")
+	err = os.MkdirAll(snykCacheDir, 0755)
```

Comment:
What does the number mean?

Reply:
https://wiki.archlinux.org/title/File_permissions_and_attributes#Numeric_method